app/styles/_core.less

```diff
@@ -1105,6 +1105,9 @@ a.disabled-link {
     }
     .add-webhook-action-btns {
         margin-top: 15px;
+        .btn-link {
+          padding-left: 0px;
```

Comment:
instead of a custom style here, just add the pad-left-none class to the link

Reply:
We might want a generic style that removes the padding from `btn-link` so we can replace `<a>` elements

Reply:
> is bootstrap putting padding on btn-link? wonder why they would do that

As far as I can it's never set explicitly to 0 and `.btn` has padding.

cc @rhamilto